app/components/header.module.css

```diff
@@ -184,3 +185,13 @@
         margin-right: var(--space-2xs);
     }
 }
+
+.sudo > button {
```

Comment:
```suggestion
.sudo-menu-item {
```

or am I missing something? 😅

Reply:
Only the 4123 refactors that occurred before I pushed the branch.